src/routes/consumer.ts

```diff
@@ -42,6 +47,8 @@ consumerRouter.get('/list', listPublishedDatasets);
 // GET /published/:datasetId
 // Returns a published dataset as a json object
 consumerRouter.get('/:dataset_id', loadPublishedDataset(), getPublishedDatasetById);
+//13fe4647-c0a2-469b-a159-7ac5bfece4e9
```

Comment:
What's this id?

Reply:
Rouge dataset ID while I was debugging and working things out.  Removed.